packages/main/src/themes/BusyIndicator.css

```diff
@@ -0,0 +1,96 @@
+:host(ui5-busy-indicator) span[data-sap-ui-wc-root] {
+	display: inline-block;
+	width: 100%;
+	height: 100%;
+}
+
+ui5-busy-indicator span [data-sap-ui-wc-root] {
+	display: inline-block;
+	width: 100%;
+	height: 100%;
+}
+
+.ui5-busy-indicator-wrapper {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+}
+
+.ui5-busy-indicator-circle {
+	display: flex;
+	background-color: var(--ui5-busy-indicator-color);
+	border-radius: 50%;
+}
+
+/* Belize specific */
+.ui5-busy-indicator-circle::before {
+	content: "";
+    width: 100%;
+    height: 100%;
+	border-radius: 100%;
+}
+
+.ui5-busy-indicator-small {
+	width: 3rem;
+    height: 1rem;
+}
+
+.ui5-busy-indicator-medium {
+	width: 5rem;
+    height: 2rem;
+}
+
+.ui5-busy-indicator-large {
+	width: 8rem;
+	height: 3rem;
+}
+
+.ui5-busy-indicator-small > .ui5-busy-indicator-circle {
+	width: 0.125rem;
+	height: 0.125rem;
+	margin: 0 0.2rem;
+}
+
+.ui5-busy-indicator-medium > .ui5-busy-indicator-circle {
+	width: 0.5rem;
+	height: 0.5rem;
+	margin: 0 0.4rem;
+}
+
+.ui5-busy-indicator-large > .ui5-busy-indicator-circle {
+	width: 1rem;
+	height: 1rem;
+	margin: 0 .75rem;
+}
+
+.circle-animation-0 {
+	animation: grow 1.6s infinite cubic-bezier(0.32, 0.06, 0.85, 1.11);
+}
+
+.circle-animation-1 {
+	animation: grow 1.6s infinite cubic-bezier(0.32, 0.06, 0.85, 1.11);
+	animation-delay: 200ms;
+}
+
+.circle-animation-2 {
+	animation: grow 1.6s infinite cubic-bezier(0.32, 0.06, 0.85, 1.11);
+	animation-delay: 400ms;
+}
+
+@keyframes grow {
+	0%, 50%, 100% {
+		-webkit-transform: scale( 1.0);
+		-moz-transform: scale( 1.0);
```

Comment:
In the old days that was indeed necessarily, but could you check if all the browser prefixed properties are still needed (we do not support opera for sure)